Makefile

```diff
@@ -1,3 +1,5 @@
+SHELL := /bin/bash # Use bash syntax
```

Comment:
Well, `SHELL := /bin/bash` already says that we're using bash syntax, probably no need to repeat that in the comment – what would be useful is to say *why*, if anything

Reply:
Fixed 👍